src/Redirectr/ApplicationBuilderExtensions.cs

```diff
@@ -11,16 +10,14 @@
 namespace Redirectr
 {
     [EditorBrowsable(EditorBrowsableState.Never)]
-    internal static class ApplicationBuilderExtensions
+    public static class ApplicationBuilderExtensions
     {
         public static IApplicationBuilder UseRedirectr(this IApplicationBuilder builder)
         {
             builder.UseEndpoints(endpoints =>
             {
                 var generator = endpoints.ServiceProvider.GetRequiredService<KeyGenerator>();
                 var options = endpoints.ServiceProvider.GetRequiredService<IOptions<RedirectrOptions>>().Value;
-                // TODO: Validate options such as, options.ShortenUrlPathTemplate is not null or empty
-                // BaseAddress exists/valid URL, ShortUrlPathPrefix is at least a slash or a valid path (/a/b/c/)
 
                 var shortenUrlPath = options.ShortenUrlPath;
```

Comment:
All this block should probably be done in a `PostConfigure`

Reply:
I got rid of this. Rather used `Url` and did some of this stuff in PostConfigure